src/libraries/Microsoft.Extensions.Logging/src/Microsoft.Extensions.Logging.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.1;netstandard2.0;net461</TargetFrameworks>
+    <IsPackable>false</IsPackable>
```

Comment:
Why Logging is not packable? You are porting this one in a separate PR?

Reply:
This one contains source generator, so we will probably needs some hacking and workarounds for making it work

Reply:
@Anipik for cases where you would expect a change to not be self-speaking could you please either add a comment inline into the sources (in this case i.e. `TODO: Figure out how to pack source generators`) or in the PR? This will help your reviewers and yourself to get better feedback and avoid unnecessary review iterations.

Reply:
Here's a sample that demonstrates how to pack an analyzer/source-generator.  No need to tackle in this PR, but might be helpful for a followup: https://github.com/ericstj/runtime/commit/ba5a7c83a5c3c62343f5c285ed52868426c2912e
cc @maryamariyan